dbmanager/dbmanager_test.go

```diff
@@ -0,0 +1,73 @@
+package dbmanager
```

Comment:
Since you rely on `init` to initialize your database tests, please make sure this is a test package, so it's not included in final builds (i.e. rename this package to `dbmanager_test`)